download-vr-cli.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - bash: |
+      set -exuo pipefail
+      npm install -g vr-approval-cli@0.0.163
```

Comment:
instead of having to have a separate template to download this, can't we just add `vr-approval-cli` as a dev dependency on the monorepo package.json?

Reply:
yes both ways are fine....Generally in 1JS, the job where this flow runs...we want to avoid checkout and yarn fast to reduce time....so we use this method.